app/components/cards/PostSummary.jsx

```diff
@@ -149,7 +149,7 @@ class PostSummary extends React.Component {
         let thumb = null;
         if(pictures && p.image_link) {
           const prox = $STM_Config.img_proxy_prefix
-          const size = (thumbSize == 'mobile') ? '640x480' : '128x256'
+          const size = (thumbSize == 'mobile') ? '640x480' : '256x384';
```

Comment:
Important to keep the aspect ratio; this may need to be `256x512` if I understand the change correctly